lmeval-evaluation-job.adoc

```diff
@@ -13,7 +13,7 @@ After the `LMEvalJob` is created, the LM-Eval service runs the evaluation job.
 
 [NOTE]
 --
-Other TrustyAI features (e.g. bias and drift metrics) do not support non-tabular models (e.g. LLMs). Deploying the TrustyAIService custom resource (CR) in a namespace that contains non-tabular models (such as the namespace where an evaluation job is being executed) can cause errors within the TrustyAI service.
+Other TrustyAI features (such as  bias and drift metrics) do not support non-tabular models (including LLMs). Deploying the TrustyAIService custom resource (CR) in a namespace that contains non-tabular models (such as the namespace where an evaluation job is being executed) can cause errors within the TrustyAI service.
```

Comment:
double-space after (such as   bias...)

Reply:
Actually, it might be just the way it's rendered here. It might not be a double-space.